server/blob/blob-api/src/main/java/org/apache/james/blob/api/MetricableBlobStore.java

```diff
@@ -70,4 +70,9 @@ public InputStream read(BucketName bucketName, BlobId blobId) {
         return metricFactory
             .runPublishingTimerMetric(READ_TIMER_NAME, () -> blobStoreImpl.read(bucketName, blobId));
     }
+
+    @Override
+    public Mono<Boolean> createBucket(BucketName bucketName) {
```

Comment:
What does that boolean means? Success? Was created vs already existed ?

I would prefer to return `Mono<Void>` rather than a 'magic' not explicit enough boolean type...

Reply:
Yeah that was the idea... However if we return `Mono<Void>`, I have no idea how to test create multiple buckets should be idempotent and is the creation successful or not... Or do I just kick out those tests?

Reply:
Well the principal of indepotence is that the result is the same wether the bucket existed before or not